docs/admin-guides/managing-optimizers.md

```diff
@@ -80,16 +80,16 @@ The following configuration needs to be filled in:
 
 - name: the name of the optimizer group, which can be seen in the list of optimizer groups on the front-end page.
 - container: the name of a container configured in containers.
-- properties: the default configuration under this group, is used as a configuration parameter for tasks when the optimize page is scaled out.
+- properties: the default configuration under this group, is used as a configuration parameter for tasks when the optimize page is scaled out.Supports native parameters for `flink on yarn`, and users can set parameters using the `<property>=<value>` or use `flink-conf.yaml` to configure parameters.
 
 The optimizer group supports the following properties:
 
-| Property            | Container type | Required | Default | Description |
-|---------------------|----------------|----------|---------|-------------|
+| Property            | Container type | Required | Default | Description                                                                                                                                                                                                                                                                                                                                                                                                      |
+|---------------------|----------------|----------|---------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | scheduling-policy   | All | No | quota | The scheduler group scheduling policy, the default value is `quota`, it will be scheduled according to the quota resources configured for each table, the larger the table quota is, the more optimizer resources it can take. There is also a configuration `balanced` that will balance the scheduling of each table, the longer the table has not been optimized, the higher the scheduling priority will be. |
-| taskmanager.memory   | flink | Yes | N/A | The memory size of Flink TaskManager. |
-| jobmanager.memory    | flink | Yes | N/A | The memory size of Flink JobManager.  |
-| memory   | local | Yes | N/A | The memory size of the local optimizer Java process. |
+| taskmanager.memory   | flink | No | N/A | The memory size of Flink TaskManager. Deprecated, suggested use `taskmanager.memory.process.size`. If not set, the parameters in `flink-conf.yaml` will be used by default.                                                                                                                                                                                                                                      |
```

Comment:
As property `taskmanager.memory` and `jobmanager.memory` is deprecated, I suggest removing them from the supported properties.

We can describe the supported properties for flink container like this:
```
| flink-conf.* | flink | No | N/A | Any configuration for `flink on yarn` mode, like `flink-conf.taskmanager.memory.process.size` or `flink-conf.jobmanager.memory.process.size`. The value in `conf/flink-conf.yaml` will be used if not setted here. You can find more supported property in [Flink Configuration](https://nightlies.apache.org/flink/flink-docs-master/docs/deployment/config/)
```

Reply:
When I try to write the document, I found we may add a prefix for the flink properties like `flink-conf`.

If we want to support some properties not used by Flink,  we can identify them then.

What do you think? @xujiangfeng001

Reply:
> https://nightlies.apache.org/flink/flink-docs-master/docs/deployment/config/

This suggestion looks good, I'll modify the logic.

Reply:
> As property `taskmanager.memory` and `jobmanager.memory` is deprecated, I suggest removing them from the supported properties.
> 
> We can describe the supported properties for flink container like this:
> 
> ```
> | flink-conf.* | flink | No | N/A | Any configuration for `flink on yarn` mode, like `flink-conf.taskmanager.memory.process.size` or `flink-conf.jobmanager.memory.process.size`. The value in `conf/flink-conf.yaml` will be used if not setted here. You can find more supported property in [Flink Configuration](https://nightlies.apache.org/flink/flink-docs-master/docs/deployment/config/)
> ```

Do we need to be compatible with `taskmanager. memory` and `jobmanager. memory` in the code here.

Reply:
Yes, we can keep the compatible code for one major version and remove it from 0.7.0.
You do better use a `@Deprecated` annotation to tell that in code.

Reply:
OKey.

Reply:
Hi @zhoujinsong , PTAL.